inference-engine/src/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -79,6 +81,8 @@ bool ngraph::pass::MOCTransformations::run_on_function(std::shared_ptr<ngraph::F
     if (!m_use_shapes) {
         manager.register_pass<ngraph::pass::DisableShapeOfConstantFolding>();
     }
+    manager.register_pass<ov::pass::RemoveConcatZeroDimInput>();
+    manager.register_pass<ov::pass::RemoveMultiSubGraphOpDanglingParams>();
```

Comment:
Move registration to NopElimination

Reply:
done

Reply:
I tested the expected model in such configuration and unfortunately has to be performed also before CT.
Let's look at the image below:
![image](https://user-images.githubusercontent.com/11331953/143413810-2ac9b870-f3ad-4c60-bd3f-d67ef498b602.png)

Reply:
discussed offline